### ISSUE TITLE
Bug: Dark Mode Contrast Does Not Meet WCAG 2.1 AA Standards

### ISSUE THREAD

**Issue by shivani-indoria** [shivani-indoria/shivani-indoria.github.io#3]
8 contrast violations detected in Dark Mode
